src/lib/core/tests/TestTLV.cpp

```diff
@@ -2789,6 +2789,52 @@ void CheckTLVByteSpan(nlTestSuite * inSuite, void * inContext)
     NL_TEST_ASSERT(inSuite, memcmp(readerSpan.data(), bytesBuffer, sizeof(bytesBuffer)) == 0);
 }
 
+void CheckTLVCharSpan(nlTestSuite * inSuite, void * inContext)
+{
+    struct CharSpanTestCase
+    {
+        const char * testString;
+        const char * expectedString;
+    };
+
+    // clang-format off
+    static CharSpanTestCase sCharSpanTestCases[] = {
+        // Test String                                                   Expected String from Get()
+        // =========================================================================================
+        {  "This is a test case #0",                                     "This is a test case #0"  },
+        {  "This is a test case #1\x1fTest Localized String Identifier", "This is a test case #1"  },
+        {  "This is a test case #2 \x1f abc \x1f def",                   "This is a test case #2 " },
```

Comment:
Please add a case that ends with IS1 alone as well

Reply:
done